pkg/command/plugin.go

```diff
@@ -202,7 +202,7 @@ func newDescribePluginCmd() *cobra.Command {
 
 func newInstallPluginCmd() *cobra.Command { //nolint:funlen
 	var installCmd = &cobra.Command{
-		Use:   "install [" + pluginNameCaps + "]",
+		Use:   "install " + pluginNameCaps,
```

Comment:
This is a bit tricky because the `PLUGIN_NAME` is optional when we use `--group`.
This was mentioned in a previous PR: https://github.com/vmware-tanzu/tanzu-cli/pull/333#discussion_r1213685505

Note that `plugin install all` is no longer supported.
I would leave this line as is.

Reply:
Maybe we should put a comment to say that the brackets are on purpose just so we don’t try to change it a third time

Reply:
> This is a bit tricky because the PLUGIN_NAME is optional when we use --group.
The current help message is below:
---------------------------------------------------- 
❯ tanzu plugin install -h
Install a specific plugin by name or specify all to install all plugins of a group
Usage:
tanzu plugin install [PLUGIN_NAME] [flags]

---------------------------------------------------- 
The PLUGIN_NAME is not optional for the `tanzu plugin install` command. Therefore, we should maintain consistency in the help message by keeping PLUGIN_NAME without square brackets. We should update the help message to include the '--group' flag.
eg:
---------------------------------------------------- 
❯ tanzu plugin install -h
Install a specific plugin by name or specify all to install all plugins of a group
Usage:
tanzu plugin install --group PLUGIN_GROUP_NAME
---------------------------------------------------- 
We can discuss with the team, and take decision.

Reply:
Actually the plugin name is optional of you use `--group` although you’re right, the help message doesn’t say that.